__tests__/app.test.js

```diff
@@ -89,4 +89,53 @@ describe('/api/articles/:article_id', () => {
             expect(response.body.msg).toBe('Bad request')
         })
     })
+})
+
+describe('/api/articles', () => {
+    test('GET:200 Should respond with an articles array of article objects', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            expect(Array.isArray(response.body)).toBe(true)
+            expect(typeof response.body).toBe('object')
+            expect(response.body).toHaveLength(13)
+        })
+    })
+    test('GET:200 Should contain the correct property keys except the body property', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body
+            articles.forEach((article) => {
+                expect(article).toHaveProperty('title')
+                expect(article).toHaveProperty('topic')
+                expect(article).toHaveProperty('author')
+                expect(article).toHaveProperty('created_at')
+                expect(article).toHaveProperty('votes')
+                expect(article).toHaveProperty('article_img_url')
+                expect(article).toHaveProperty('comment_count')
+                expect(article).not.toHaveProperty('body')
+            })
+        })
+    })
+    test('GET:200 Articles should be sorted by date in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body
+            expect(articles[0].created_at).toContain('2020-11-03')
+            expect(articles[12].created_at).toContain('2020-01-07')
+        })
+    })
+    test('GET:404 Should respond with an appropiate status and error message when given a valid but not existant endpoint', () => {
```

Comment:
This is a valid test, but shouldn't be in this describe block. It's not pertaining to articles but all malformed endpoints